### ISSUE TITLE
Request: Upload machine definitions

### ISSUE THREAD

**Issue by ospalh** [markwal/OctoPrint-GPX#40]
Looking at the GPX (w/o OctoPrint ) Readme, you can use `-c` to use a new machine definition.
I have found just a file like that for my printer, a Makerbot clone from CTC, and can’t tell the plugin to use it, but have to modify an existing file, which is error-prone.

**Comment by markwal** [comment]
True.  The plugin doesn't have a feature for uploading an .ini file.

Changing the title to reflect a feature request.

**Comment by markwal** [comment]
BTW, I believe the CTC is a clone of a Replicator 1 Dual, so you can also just choose that from the drop down.

**Comment by ospalh** [comment]
The point is that it’s a Replicator 1 clone, with *different* stepper and extruder values.
I selected the Replicator 1 and had to change the values. Works, but could be simpler.
